docs/disasterRecovery/disasterRecovery.md

```diff
@@ -0,0 +1,74 @@
+# CCBC application disaster recovery
+
+In case of a database corruption, crash or any other issue requiring us to restore the database to a previous state, the OpenShift `PostgresCluster` object allows us to restore the database from any point in time prior to the incident.
+
+---
+
+## Makefile target
+
+**Prerequisites:**
+
+- The ENVIRONMENT (dev,test,prod) and CIF_NAMESPACE_PREFIX, GGIRCS_NAMESPACE_PREFIX, CIIP_NAMESPACE_PREFIX environment variables are set
+- The machine is logged in onto the cluster where the CIF app is deployed (`oc login ...`)
+
+**Restoring the DB:**
+
+To restore the database to any point in time, simply run:
+
+```
+make restore TARGET_TIMESTAMP="2022-05-24 14:20:00-07"
```

Comment:
We don't have a `make restore` command in our makefile, would be valuable to add this in or update this with the manual steps